docker-compose.yml

```diff
@@ -56,8 +56,18 @@ services:
       - db
       - redis-store
       - redis-cache
+    volumes:
+      - $PWD:/app
     command: test
 
+  test-ci:
+    extends:
+      service: base
+    links:
+      - db
+      - redis-store
+      - redis-cache
+
```

Comment:
I tried to be "neater" here to not have to list the `links: ...` two times but docker-compose doesn't allow that. Now I violate the DRY principle but it's fine. 
Note also, that there's no need for a `command` in the `test-ci` directive.

Reply:
The explicit links are not necessary: https://docs.docker.com/compose/networking/#links